tests/test_commands.py

```diff
@@ -0,0 +1,78 @@
+from datetime import timedelta
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils import timezone
+
+from django_future.jobs import schedule_job
+from django_future.models import ScheduledJob
+
+
+class RunScheduledJobsCommandTest(TestCase):
+
+    def setUp(self):
+        self.schedule_at = timezone.now() - timedelta(days=1)
+
+        self.jobs = [
+            schedule_job(self.schedule_at, 'math.pow', args=(2, 3)),
+            schedule_job(self.schedule_at, 'math.pow', args=(5, 2))
+        ]
+
+    def test_cmd_noargs(self):
+        """
+        Test invocation of command with no arguments. Ensure the scheduled jobs
+        are marked as completed.
+        """
+        self.assertEqual(
+            2,
+            ScheduledJob.objects.filter(
+                status=ScheduledJob.STATUS_SCHEDULED).count()
+        )
```

Comment:
I like these (redundant) checks in all the test methods. Makes for good self-documented code.